pkg/cron/cron.go

```diff
@@ -93,6 +97,13 @@ func (c *Cron) requestCall() error {
 	}
 	if etag != "" {
 		c.etag = etag
+		configmap := &corev1.ConfigMap{
```

Comment:
I think we should also check if the config map exists here first just in case it was deleted we can recreate. Maybe we can create a separate function which is used by both cron and the controller run function which gets the current configmap and then does a create / update? We can also have a simple unit test for that function as well.